dvc/command/get.py

```diff
@@ -10,7 +10,27 @@
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _show_url(self):
+        from dvc.api import get_url
+
+        try:
+            url = get_url(
+                self.args.path, repo=self.args.url, rev=self.args.rev
+            )
+            logger.info(url)
+        except DvcException:
+            logger.exception("failed to show url")
```

Comment:
Capitalization: "Failed to show URL"
Maybe end in period `.` ? (This depends on whether it's printed as part of a larger message – may also apply to capitalizing first letter `f`.

Reply:
@jorgeorpinel, the lowercase seems to be used everywhere in the codebase. And, because this is a `logger.exception`, our formatter handles adding extra information from raised exception besides this particular logging message. Notice the messages on following examples:
```sh
➜ dvc get https://github.com/schacon/cowsay install.sh --show-url 
ERROR: failed to show url - URL 'https://github.com/schacon/cowsay' is not a dvc repository.
➜ dvc get . dir/file --show-url
ERROR: failed to show url - unable to find DVC-file with output 'dir/file'
```

Reply:
`failed to show url - URL` it looks strange though :) First it proves that `the lowercase seems to be used everywhere` is not true (and it should not be, typically, acronyms and initialisms are written in all capital letters to distinguish them from ordinary words. - NASA, URL, DVC, etc). Second, you have a repetition here. Repetition is probably fine if reasons vary and you can't actually control them.

Reply:
@shcheklein, it's not something that I add, but, I'm utilizing the fact that something will be there. So, it's more like "(what failed) - (the reason)".

Reply:
And, yes, regarding lowercase, I just looked into it and it's not everywhere for other cases, but for `exceptions`, lowercase is used *everywhere*.

Reply:
> because this is a logger.exception, our formatter handles adding extra information

Thanks, very useful to know this.

> lowercase seems to be used everywhere is not true...

➕ 1️⃣  No need to use bad grammar on purpose because of existing bad grammar.

> >failed to show url - URL it looks strange though :) ...
>
> it's more like "(what failed) - (the reason)".

So then `the reason` part in this case should start directly with `http://...` – but that's not part of this PR...

Reply:
> the reason part in this case should start directly with http://... – but that's not part of this PR 

OK I changed this in https://github.com/iterative/dvc/pull/3220/commits/ec0c39a28ec8a26b76843a715449d54d810fcbb3#diff-3b2745249f7139953ab4b45b382e69e9 but this whole conversation kind of opened a can of worms and that PR grew quite a bit 😅  Hoping core team doesn't flip out.

> but for exceptions, lowercase is used everywhere

Thanks for the tip. I may review some of these too 😋

Reply:
p.s. that PR = iterative/dvc/pull/3220